core/src/main/java/org/apache/carbondata/core/metadata/schema/PartitionInfo.java

```diff
@@ -24,40 +24,29 @@
 import org.apache.carbondata.core.metadata.schema.table.column.ColumnSchema;
 
 /**
- * Partition Information of carbon partition table
+ * Partition information of carbon partition table
  */
 public class PartitionInfo implements Serializable {
 
-  /**
-   * Partition columns
-   */
   private List<ColumnSchema> columnSchemaList;
 
-  /**
-   * partition type
-   */
   private PartitionType partitionType;
 
   /**
-   * Range Partition definition
+   * range infomation defined for range partition table
    */
   private List<String> rangeInfo;
 
   /**
-   * List Partition definition
+   * value list defined for list partition table
    */
   private List<List<String>> listInfo;
 
   /**
-   * Hash Partition numbers
+   * hash partition numbers
```

Comment:
This should be number of partition. 
The variable name suggest to change to `numPartitions`, which is aligned with spark